### DISCUSSION TITLE
YOLO support in frigate

### DISCUSSION THREAD

**Discussion by hridaynathp** [blakeblackshear/frigate#9290] [Ask A Question] [answered]
Hi,
Recently I came across the YOLO webpage which mentioned about commercialization of YOLO.
Now there are two options available 1. AGPL (free) 2. Enterprise.
I have question now, How frigate is going to use YOLO?
Will AGPL fulfill the requirements of frigate? or frigate will phase out YOLO and will switch to other similar models?
Please help here to understand the future engagement with YOLO.

Thanks,
Hridaynath

**Comment by blakeblackshear** [comment, marked as answer]
Update: https://github.com/blakeblackshear/frigate/pull/10717

~Frigate is open source and can use freely available YOLO models for some detector types. Users can also bring their own models. I don't believe the way Frigate is currently supporting the use of YOLO models requires an Enterprise license. There are no plans to "phase out" support for YOLO models.~
